src/commands/releases.rs

```diff
@@ -108,6 +108,10 @@ pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
                 .help("When the flag is set and the previous release commit was not found in the repository, \
                         will create a release with the default commits count (or the one specified with `--initial-depth`) \
                         instead of failing the command."))
+            .arg(Arg::with_name("allow-empty")
+                .long("allow-empty")
```

Comment:
Not sure if `allow-empty` is the correct phrase to use here since `--allow-empty` in git adds commits with no content in them. In this case, this option refers to the fact that there is no commits to add to a release - I'm sure someone will get this confused when they try and use the `allow-empty` option in it's current state.

Maybe we should use `ignore-empty`. It won't conflict with git's `--allow-empty` (which may prevent future user misunderstandings), and `ignore` matches what is happening better than `allow` (updating the release is ignored).

Reply:
Valid reasoning. Changed